src/planwise/endpoint/home.clj

```diff
@@ -3,6 +3,7 @@
             [ring.util.anti-forgery :refer [anti-forgery-field]]
             [buddy.auth :refer [authenticated? throw-unauthorized]]
             [planwise.component.auth :refer [create-jwe-token]]
+            [cheshire.core :as json]
```

Comment:
Mostly for consistency. `wrap-json-params` and `wrap-json-response` use Cheshire already. Also it's faster: https://github.com/dakrone/cheshire#why :)